packages/expo-cli/src/commands/build/ios/appleApi/authenticate.js

```diff
@@ -40,7 +40,8 @@ function _getAppleIdFromParams({ appleId }) {
   }
 }
 
-async function _promptForAppleId({ appleId }) {
+async function _promptForAppleId(options) {
+  let appleId = options.appleId;
```

Comment:
Why did you move it here?

Reply:
I don't remember, I guess I needed the `options` object in some iteration of this. I've reverted this change.